examples/erp/genseki/config.tsx

```diff
@@ -84,6 +95,14 @@ const app = new GensekiApp({
       },
     },
   })
+  .apply({
+    api: {
+      storage: {
+        putObjSignedUrl: uploadHandlers['file.generatePutObjSignedUrl'],
+        getObjSignedUrl: uploadHandlers['file.generateGetObjSignedUrl'],
+      },
+    },
+  })
```

Comment:
don't do this. The handler should be created when storageAdapter is provided

Reply:
Good but prefer something like this (For convention).

```tsx
const storageAdapter = new StorageAdapterS3(context, {
  // ...
})
```

The key is bringing `context` as a first args